### PR TITLE
Private phase 1 (and some 2) instructions

### LINKED ISSUE #104: Update instructions for Private: Phase 1

### PR DESCRIPTION
**Pull request by IAlibay**
Fixes #104 

## Licensing agreement
* [x] I agree to release these inputs under the combined MIT and CCBY SA 4.0 licenses of this repository

### CODE REVIEW

File: docs/source/private/overview.rst [hunk 1]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+===================
+
+How the datasets are prepared is fully up to you. We encourage that use lessons learnt from the
+public set :ref:`input preparation step <input-preparation>` as a guide on how to prepare systems
+for use with OpenFE. We expect that you will have a PDB for your protein, an SDF for your ligands,
+and optionally another SDF for your cofactors.
+
+The only additional requirements for dataset preparations are:
+* Any ligand names should be anonymised. The OpenFE data gathering scripts will assume names to be anonymous.
+* If possible, you record any methods used in dataset preparation, as suitable for the SI of a journal publication.
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:76]: Maybe we can encourage them to also run the `input_validation.py` script to check if their proteins are ready for OpenFE?

File: docs/source/private/overview.rst [hunk 2]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+
+.. note::
+   Please keep all post-cleanup data around for analysis until the end of the benchmarks (i.e.
+   after publication).
+
+
+The OpenFE tools are known to generate a lot of data by default (something we are looking to fix!).
+
+We recommend that folks use the :ref:`simulation cleanup <post-simulation cleanup>` script to clean up
+unecessary data.
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:149]:
```suggestion
unnecessary data.
```

File: docs/source/private/overview.rst [hunk 3]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+
+
+Inspecting Results
+==================
+
+
+.. note::
+   A separate script will be provided for gathering relevant FE output data in Phase 2.
+   This is not how all the data that OpenFE will gather.
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:158]: I don't know what this sentence means, could you rephrase?

File: docs/source/private/overview.rst [hunk 4]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+* Number of completed & failed edges.
+  * Including information about any attempts at remediating broken networks due to failed edges.
+* Simulation time and the number of atoms.
+* Blinded transformation networks (i.e. name of nodes and how they are connected).
+* Transformation information.
+  * Mapping scores (Lomap, formal charge, shape overlay, RMSD, number of heavy dummy & core atoms) for each edge.
+  * 2D and 3D fingerprint similarity scores (e.g. Tanimoto).
+  * Number of rotatable bonds & rings.
+
+Optionally (if you can't that's completely ok!) we would also the following:
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:215]: A verb seems to be missing in this sentence

File: docs/source/private/prep_and_debug.rst [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+
+This FAQ contains a list of issues you may encounter when preparing and
+running simulations with OpenFE. This list will be updated as feedback
+is provided by benchmark partners.
+
+
+1. Ligand Alignment
+*******************
+
+Ligands in a series should be pre-aligned when pass to OpenFE tools.
```
Comment (hannahbaumann) [inline docs/source/private/prep_and_debug.rst:15]:
```suggestion
Ligands in a series should be pre-aligned when passed to OpenFE tools.
```

File: docs/source/private/overview.rst [hunk 6]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+
+Dataset Preparation
+===================
+
+How the datasets are prepared is fully up to you. We encourage that use lessons learnt from the
+public set :ref:`input preparation step <input-preparation>` as a guide on how to prepare systems
+for use with OpenFE. We expect that you will have a PDB for your protein, an SDF for your ligands,
+and optionally another SDF for your cofactors.
+
+The only additional requirements for dataset preparations are:
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:74]:
```suggestion
The only additional requirements for dataset preparations are:

```
Adding an empty line for the bullet points.

File: docs/source/private/prep_and_debug.rst [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+
+3. All My Complex Edges Fail But Not The Ligands
+************************************************
+
+This indicates a likely issue with the protein structure, either a clash
+with the ligand, or a clash within the protein structure itself.
+
+To check this we suggest running a conventional MD simulation and seeing if the system remains stable.
+You can use the `OpenFE MD Protocol for this purpose <https://docs.openfree.energy/en/stable/tutorials/md_tutorial.html>`.
```
Comment (hannahbaumann) [inline docs/source/private/prep_and_debug.rst:42]:
```suggestion
You can use the `OpenFE MD Protocol for this purpose <https://docs.openfree.energy/en/stable/tutorials/md_tutorial.html>`_.
```

File: docs/source/private/prep_and_debug.rst [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+
+4. My Ligand Transformations Are Failing
+****************************************
+
+This likely indicates a bad mapping.
+
+We would encourage you to `visualize your mappings <https://docs.openfree.energy/en/stable/cookbook/ligandnetwork_vis.html>`.
```
Comment (hannahbaumann) [inline docs/source/private/prep_and_debug.rst:50]:
```suggestion
We would encourage you to `visualize your mappings <https://docs.openfree.energy/en/stable/cookbook/ligandnetwork_vis.html>`_.
```

Comment (hannahbaumann) [reply docs/source/private/prep_and_debug.rst:50]: This cookbook used as example an MST and LomapAtomMapper, it might be nice to have an example on how to load in the `ligand_network.graphml` and look at the mapping from there, but may also not be required.

File: docs/source/private/overview.rst [hunk 9]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+* Calculated ddG values.
+* Experimental assay values (micromolar).
+* Broad assay description (e.g. Kd from ITC).
+* Estimated benchmark difficulty (i.e. "easy", "medium", "hard").
+* Convergence data (MBAR overlap, forward/backwards plots, etc...).
+* Number of completed & failed edges.
+  * Including information about any attempts at remediating broken networks due to failed edges.
+* Simulation time and the number of atoms.
+* Blinded transformation networks (i.e. name of nodes and how they are connected).
+* Transformation information.
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:210]:
```suggestion
* Transformation information.

```
Extra line for bullet points below

**Review by hannahbaumann — CHANGES_REQUESTED** [review]
Thanks @IAlibay , just left a few small comments, otherwise lgtm!

File: docs/source/private/overview.rst [hunk 10]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+Dataset Preparation
+===================
+
+How the datasets are prepared is fully up to you. We encourage that use lessons learnt from the
+public set :ref:`input preparation step <input-preparation>` as a guide on how to prepare systems
+for use with OpenFE. We expect that you will have a PDB for your protein, an SDF for your ligands,
+and optionally another SDF for your cofactors.
+
+The only additional requirements for dataset preparations are:
+* Any ligand names should be anonymised. The OpenFE data gathering scripts will assume names to be anonymous.
```
Comment (jameseastwood) [inline docs/source/private/overview.rst:75]: How about ". . . The OpenFE data gathering scripts will not modify the existing names."

File: docs/source/private/overview.rst [hunk 11]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+.. code-block:: bash
+
+   wget https://raw.githubusercontent.com/OpenFreeEnergy/IndustryBenchmarks2024/main/industry_benchmarks/utils/extras/extract_results.py
+   python extract_results.py
+
+
+This will provide both dG and ddG outputs for you to further manipulate.
+
+As we cannot tell what format your experimental results are in, we do not provide a plotting script at this time
+and encourage you to use your own internal plotting tools. Note: this may change depending on demand.
```
Comment (jameseastwood) [inline docs/source/private/overview.rst:174]: Maybe we shouldn't encourage them to request more scripts?

File: docs/source/private/overview.rst [hunk 12]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+
+You are encouraged to share early results with everyone on the #industry-benchmarking slack channel!
+
+
+Handling Failed Edges
+=====================
+
+
+.. note::
+   Please keep a note of any failed edges, this should be reported on results submission.
```
Comment (jameseastwood) [inline docs/source/private/overview.rst:185]: How about "...failed edges and report them when you submit results."

File: docs/source/private/overview.rst [hunk 13]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+
+
+You should handle failed edges in the same way as the :ref:`public datasets <failed_edges>`.
+
+A script will be provided to try to fix broken networks soon.
```
Comment (jameseastwood) [inline docs/source/private/overview.rst:190]: "A script to try to fix broken networks will soon be provided."

File: docs/source/private/overview.rst [hunk 14]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+***********************
+
+What data will we gather?
+=========================
+
+We will need the following data from you:
+
+* Calculated ddG values.
+* Experimental assay values (micromolar).
+* Broad assay description (e.g. Kd from ITC).
```
Comment (jameseastwood) [inline docs/source/private/overview.rst:203]: add quotation marks to "Kd from ITC"

File: docs/source/private/overview.rst [hunk 15]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+* Estimated benchmark difficulty (i.e. "easy", "medium", "hard").
+* Convergence data (MBAR overlap, forward/backwards plots, etc...).
+* Number of completed & failed edges.
+  * Including information about any attempts at remediating broken networks due to failed edges.
```
Comment (jameseastwood) [inline docs/source/private/overview.rst:207]: bullet indent is not working, and I don't remember my markdown well enough to fix it offhand

File: docs/source/private/overview.rst [hunk 16]
```diff
@@ -27,3 +22,212 @@ Here each industry partner will be expected to:
...
+* Simulation time and the number of atoms.
+* Blinded transformation networks (i.e. name of nodes and how they are connected).
+* Transformation information.
+  * Mapping scores (Lomap, formal charge, shape overlay, RMSD, number of heavy dummy & core atoms) for each edge.
+  * 2D and 3D fingerprint similarity scores (e.g. Tanimoto).
```
Comment (jameseastwood) [inline docs/source/private/overview.rst:212]: Are we collecting Tanimoto similarity, or something else? No one has proposed any alternative, or complained very loudly about Tanimoto, so I think we should just go with that.

File: docs/source/private/prep_and_debug.rst (hunk 7 above)
Comment (jameseastwood) [reply docs/source/private/prep_and_debug.rst:42]: I'm not sure what's going on with the formatting here, either in the original or in Hannah's suggested change.

Comment (jameseastwood) [reply docs/source/private/prep_and_debug.rst:42]: Maybe it's a broken hyperlink?

**Review by jameseastwood — CHANGES_REQUESTED** [review]
Most of this is minor copy editing, but there are a couple of points where we need to clean up the details of what we're communicating to the partners.

File: docs/source/private/prep_and_debug.rst (hunk 8 above)
Comment (IAlibay) [reply docs/source/private/prep_and_debug.rst:50]: I unfortunately do not have the time to make one, is this something you might have time for?

File: docs/source/private/overview.rst (hunk 1 above)
Comment (IAlibay) [reply docs/source/private/overview.rst:76]: Done above.

File: docs/source/private/overview.rst (hunk 11 above)
Comment (IAlibay) [reply docs/source/private/overview.rst:174]: Removed.

File: docs/source/private/overview.rst (hunk 12 above)
Comment (IAlibay) [reply docs/source/private/overview.rst:185]: Done.

**Comment by IAlibay** [comment]
@hannahbaumann @jameseastwood please have another look, all your comments should have been addressed.

File: docs/source/private/prep_and_debug.rst (hunk 8 above)
Comment (hannahbaumann) [reply docs/source/private/prep_and_debug.rst]: I just checked, I have an example notebook for loading in the transformation .json file and displaying a mapping (the one we sent to Christina a while ago). Would that be helpful?

File: docs/source/private/overview.rst [hunk 17]
```diff
@@ -27,3 +22,225 @@ Here each industry partner will be expected to:
...
+* Ligand series that undergo cyclisation.
+* Ligand series that undergo scaffold hopping (i.e. very small to no conserved core).
+
+Should you end up using a system with known challenges, we would ask you to describe the challenging nature of the dataset upon submission.
+
+
+Dataset Preparation
+===================
+
+How the datasets are prepared is fully up to you. We encourage that use lessons learnt from the
```
Comment (hannahbaumann) [inline docs/source/private/overview.rst:69]:
```suggestion
How the datasets are prepared is fully up to you. We encourage that you use lessons learnt from the
```

**Review by hannahbaumann — COMMENTED** [review]
Lgtm, just the small comment!

**Review by hannahbaumann — APPROVED** [review]

**Review by jameseastwood — APPROVED** [review]
Thank you!
